chapters/equations.tex

```diff
@@ -817,13 +817,19 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 variables) is equal.  Often, this is a nonlinear system of equations and therefore it may be necessary to provide appropriate guess values (i.e., \lstinline!start! values and
 \lstinline!fixed = false!) in order to compute a solution numerically.
 
-It may be difficult for a user to figure out how many initial equations have to be added, especially if the system has a higher index. A tool may add or remove initial equations
-automatically such that the resulting system is structurally nonsingular.  In these cases diagnostics are appropriate since the result is not unique and not necessarily what the user
-expects.  A missing initial value of a discrete-time variable which does not influence the simulation result, may be automatically set to the \lstinline!start! value or its default without
-informing the user.  For example, variables assigned in a \lstinline!when!-clause which are not accessed outside of the \lstinline!when!-clause and where \lstinline!pre! is not explicitly
-used on these variables, do not have an effect on the simulation.
+It may be difficult for a user to figure out how many initial equations have to be added, especially if the system has a higher index.
 \end{nonnormative}
 
+Based on these non-normative considerations we have the following rules:
+A tool may add or remove initial equations automatically according to the rules below such that the resulting system is structurally nonsingular.
+\begin{itemize}
+\item A missing initial value of a discrete-time variable (see \cref{component-variability-prefixes-discrete-parameter-constant} - this does not include parameter and constant variables), which does not influence the simulation result, may be automatically set to the start value or its default without informing the user.
+For example, variables assigned in a \lstinline!when!-clause which are not accessed outside of the \lstinline!when!-clause and where \lstinline!pre! is not explicitly used on these variables, do not have an effect on the simulation.
+\item A \lstinline!start!-attribute that is not fixed may be treated as fixed with a diagnostic.
+\item A consistent start value or initial equation may be removed with a diagnostic.
+\end{itemize}
+Tools must not change the initialization in a way that does not satisfy the initial equations and fixed start values.
```

Comment:
Proper paragraph break:
```suggestion
\end{itemize}

Tools must not change the initialization in a way that does not satisfy the initial equations and fixed start values.
```

Reply:
Am I missing the reason why this needs to be normative?  If it is a rule meant particularly for the turning of non-fixed to fixed, maybe this could be made more clear (maybe put it back in the list, but as a sub-item)?  It's already stated that these rules can only be applied to make the initialization structurally non-singular – isn't that enough?

Reply:
Maybe this was resolved by mistake?  I couldn't see any sign of a resolution or reason to resolve without action.

Maybe the problem was that two different comments were grouped into the same conversation – I suppose this is a GitHub limitation one has to get used to?

Reply:
Ok, making it non-normative and re-formulating.
The reason for re-formulating is that "must not" is a strict requirement, and it doesn't make sense to have that as non-normative text.

Reply:
Looks OK now.  Resolving.